packages/main/src/plugin/kubernetes-client.spec.ts

```diff
@@ -54,6 +56,17 @@ test('Create Kubernetes resources with v1 resource should return ok', async () =
   expect(spy).toBeCalled();
 });
 
+test('Create Kubernetes resources with apps/v1 resource should return ok', async () => {
+  const client = new KubernetesClient({} as ApiSenderType, configurationRegistry, fileSystemMonitoring);
+  const createNamespacedDeploymentMock = vi.fn();
+  makeApiClientMock.mockReturnValue({
+    createNamespacedDeployment: createNamespacedDeploymentMock,
+  });
+
+  await client.createResources('dummy', [{ apiVersion: 'apps/v1', kind: 'Deployment' }]);
+  expect(createNamespacedDeploymentMock).toBeCalledWith('default', { apiVersion: 'apps/v1', kind: 'Deployment' });
+});
```

Comment:
Should we add a test for Pods as well? As that's what's mainly going to be deploying with closing https://github.com/containers/podman-desktop/issues/1261